### ISSUE TITLE
[ACCESSIBILITY] App content should not restrict its view and operation to single display orientation

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1457]
Describe the accessibility issue
[WCAG 1.3.4 Orientation](https://www.w3.org/TR/WCAG21/#orientation) states that " Content does not restrict its view and operation to a single display orientation, such as portrait or landscape, unless a specific display orientation is [[essential](https://www.w3.org/TR/WCAG21/#dfn-essential)]"

The app is designed to only be one view, portrait thus rendering not in compliance with this AA level guideline. 
Some users of assistive technology lock their viewports and therefore this guideline provides that the orientation should not be constricted to single display orientation. 

Screenshots
entire app

Smartphone (please complete the following information):
 iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Comment by acovher2** [comment]
I am well aware this app has been this way since the beginning of time, however, it is my duty to report and make the organization aware of violations existing as we could be liable for the inaccessibility of any products we are associated with. I understand this is more than a bug, but an entire redesign of app so just opening for the powers that be to discuss. 

:) 
Thanks

**Comment by acovher2** [comment]
@pmarkhennessy Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant. This was on the PPT I sent you.

**Comment by pmarkhennessy** [comment]
@acovher2 Yes I am aware and will assign it during or after the holiday

**Comment by acovher2** [comment]
@pmarkhennessy Thank you, I just have some other projects that are about to be taking priority so I wanted to ensure these were still on someone's radar and in progress. Happy new year!

**Comment by acovher2** [comment]
Verified existing in 6.1.66

**Comment by acovher2** [comment]
This exists in 7.0.14 version of mobile app. 

This is resolved in 7.0.5 version on UIUC 7.0 Dev Web
https://api-dev.rokwire.illinois.edu/illinois/ HOWEVER, this is completely inaccessible by keyboard navigation (will open other bugs for this in future with evaluation report.)

**Comment by TodorBachvarov** [comment]
@acovher2, Landscape orientation was enabled in version 7.2.0

**Comment by acovher2** [comment]
Verified as fixed in 7.2.22 for iPhone.
